packages/audit/scripts/dashboard-html.py

```
@@ -0,0 +1,470 @@
+    COLORS["red"],
+]
+
+
+# ---------------------------------------------------------------------------
+# Safe stat helpers — never raise
+# ---------------------------------------------------------------------------
+def _safe_calc(fn: Callable[[list[dict]], dict], events: list[dict]) -> dict:
+    """Call a calc function and return empty dict on failure."""
+    try:
+        return fn(events)
+    except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)